doc/3/controllers/realtime/subscribe/index.md

```diff
@@ -52,10 +52,3 @@ _Subscription to document notifications with scope option_
 
 <<< ./snippets/document-notifications-leave-scope.java
 
-_Subscription to message notifications_
```

Comment:
Why this change?
I see nothing about it in the PR description, and I don't get why you deactivate those snippets.

Reply:
Because those snippets does not exists, I should add it in the PR description in deed